packages/apps/crm/src/screens/lead/LeadDetailsScreen.js

```diff
@@ -23,21 +23,33 @@ import {
   NotesCard,
   ScrollView,
 } from '@axelor/aos-mobile-ui';
-import {useSelector, useDispatch, useTranslator} from '@axelor/aos-mobile-core';
+import {
+  useSelector,
+  useDispatch,
+  useTranslator,
+  useIsFocused,
+} from '@axelor/aos-mobile-core';
 import {LeadHeader, LeadDropdownCards, LeadBottom} from '../../components';
 import {fetchLeadById} from '../../features/leadSlice';
 
 const LeadDetailsScreen = ({route}) => {
   const {idLead, versionLead, colorIndex} = route.params;
   const I18n = useTranslator();
   const dispatch = useDispatch();
+  const isFocused = useIsFocused();
 
   const {loadingLead, lead} = useSelector(state => state.lead);
 
   const getLead = useCallback(() => {
     dispatch(fetchLeadById({leadId: idLead}));
   }, [dispatch, idLead]);
 
+  useEffect(() => {
+    if (isFocused) {
+      getLead();
+    }
+  }, [getLead, isFocused]);
+
   useEffect(() => {
     getLead();
   }, [getLead]);
```

Comment:
I think this is not required anymore as there is an useEffect with isFocused that is doing the same thing, can you check pelase ? If it's working please refactor on the other screens